xapian-letor/tests/api_letor.cc

```diff
@@ -359,6 +359,18 @@ DEFINE_TESTCASE(preparetrainingfileonedb, generated)
     return true;
 }
 
+// test whether query id's are unique in queryfile.
```

Comment:
"query ids" is generally preferred by English style guides.

Reply:
fixed in a fixup commit.